lib/theming/src/utils.ts

```diff
@@ -1 +1,26 @@
+import { rgba, lighten, darken } from 'polished';
+
 export const mkColor = (color: string) => ({ color });
+
+// Passing arguments that can't be converted to RGB such as linear-gradient
+// to library polished's functions such as lighten or darken throws the error
+// that crashes the entire storybook. It needs to be guarded when arguments
+// of those functions are from user input.
+const isLinearGradient = (color: string) => {
+  return typeof color === 'string' && color.includes('linear-gradient');
```

Comment:
I think this function is too specific, linear-gradients aren't the only type of gradients.

How about:

```js
const isColorVarChangeable = (color: string) => {
  return !!color.match(/(gradient|var)/);
}
```

related to #6135

Reply:
Oh, you are right... I only thought about the linear-gradient even though I wrote a comment `things that can't be converted to RGB`  I change to your function! Please check

Reply:
Could you add a few test cases for the function?

Reply:
@ndelangen I wrote some tests and it worked fine, but I'm not sure about updating [the snapshot test](https://circleci.com/gh/storybookjs/storybook/130895?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link)